### ISSUE TITLE
sync: reports "Updating tests..." when it's not

### ISSUE THREAD

**Issue by wolf99** [exercism/configlet#622]
Noticed that when using `sync` to update *documents*, the output indicates that configlet is updating tests:

```console
~/git/c[sync]❯ ./bin/configlet --version
4.0.0-beta.4

~/git/c[sync]❯ ./bin/configlet sync -o -e crypto-square -u
Checking exercises...
[warn] docs: instructions unsynced: crypto-square
sync the above docs ([y]es/[n]o)? y
Updated the docs for 1 Practice Exercise
Updating tests...                                                                    # No its not!
The `crypto-square` exercise has up-to-date docs, filepaths, metadata, and tests!
```

Perhaps this refers to some tests other than those in the exercise?
In which case probably a re-wording of the statement could be useful.

Thanks!

**Comment by ee7** [comment]
Thanks for the opening this issue.

TL;DR: I'll suggest removing the message

> Updating tests...

---

I'll try to explain. You write

> Noticed that when using `sync` to update _documents_

but the given command operates on the entire syncing scope (docs, filepaths, metadata, tests):

```shell
./bin/configlet sync -o -e crypto-square -u
```

as no option to restrict the syncing scope is provided. So it's correct that configlet looks at the tests for the `crypto-square` exercise.

The "Updating tests..." message is trying to say "OK, we've checked docs, filepaths, and metadata for `crypto-square` - we'll start checking its tests now, and if we find anything, we'll prompt to update". But because all the tests for `crypto-square` are up to date, there is no further output to say "Do you want to include this test case?".

That is, the answer is "no" to:

> Perhaps this refers to some tests other than those in the exercise?

And adding the `--docs` option, we don't see the "Updating tests..." message:

```console
$ ./bin/configlet sync -o -e crypto-square -u --docs
Checking exercises...
[warn] docs: instructions unsynced: crypto-square
sync the above docs ([y]es/[n]o)? y
Updated the docs for 1 Practice Exercise
The `crypto-square` exercise has up-to-date docs!
```

But I agree it's confusing, partly because configlet doesn't log things like:

```text
Checking docs...
Checking filepaths...
Checking metadata...
```

as it enters the other syncing scopes. This is a leftover from the `canoncial_data_syncer` days when the application only did syncing, was destructive by default (we had `--check`, not `--update`), and only operated on tests. 

So I think the options are:

1. Remove the "Updating tests..." message.
2. Change it to "Checking tests...", and add a "Checking foo..." message for each of the other syncing scopes

But I think option 2 is probably too verbose, at least for the default verbosity level. So I'll create a PR for option 1.

Does that help?
